src/makeRobot.js

```diff
@@ -37,8 +37,60 @@
  *
  * @return {Robot}
  */
+const EVACUATE_POSITION_X = 1400;
+const EVACUATE_POSITION_Y = 500;
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(steps = 1) {
+      this.coords.y += Number(steps) > 0 ? steps : 0;
```

Comment:
that also work but I think like that is cleaner: 
```suggestion
    if(steps > 0) {
        this.coords.y += steps
      }
```

Reply:
done with ifs :P